stage1/reaper/reaper.sh

```diff
@@ -7,5 +7,8 @@ if [ $# -eq 1 ]; then
     app=$1
     status=$(${SYSCTL} show --property ExecMainStatus "${app}.service")
     echo "${status#*=}" > "/rkt/status/$app"
+    if [ "${status#*=}" != 0 ] ; then
+        ${SYSCTL} exit ${status#*=}
+    fi
```

Comment:
Can you add a comment?
```
# The command "systemctl exit $status" is available since systemd-v227. On older
# systemd, the command will fail and rkt will just exit with return code 0.
```